src/custom-types.json

```diff
@@ -652,7 +652,12 @@
     ]
   },
   "image/pjpeg": {
-    "compressible": false
+    "compressible": false,
+    "extensions": ["jfif"],
+    "sources": [
+      "https://github.com/microsoft/referencesource/blob/master/System.Web/MimeMapping.cs#L212",
```

Comment:
you should use permalinks, otherwise those links will be wrong / invalid some time in future

Reply:
made the changes